edb/server/protocol/auth_ext/http.py

```diff
@@ -1219,14 +1301,15 @@ async def handle_magic_link_email(
             if redirect_on_failure is None:
                 raise ex
             else:
-                if not self._is_url_allowed(redirect_on_failure):
-                    raise errors.InvalidData(
-                        "Redirect URL does not match any allowed URLs.",
-                    )
```

Comment:
Is this intentionally removed?

Reply:
oof, good catch. I'll do another audit to make sure there aren't more unintentional omissions like this.

Reply:
Well... That escalated quickly: #7966